app/components/modals/speakerimg-cropper-modal.js

```diff
@@ -0,0 +1,42 @@
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend({
+  onVisible() {
```

Comment:
No need for this file

Reply:
@kushthedude , how to get 300x300 image by that same cropper which is giving a 2:1 image.

Reply:
And Also we can't change the viewport size too.

Reply:
observe the route of page and use an if conditional for view port